src/drc/api/viewsets.py

```diff
@@ -460,3 +489,25 @@ class EnkelvoudigInformatieObjectAuditTrailViewSet(AuditTrailViewSet):
     Een specifieke audit trail regel opvragen.
     """
     main_resource_lookup_field = 'enkelvoudiginformatieobject_uuid'
+
+
+class BestandsDeelViewSet(NotificationUpdateMixin,
+                        # AuditTrailViewSet,
```

Comment:
my linter is complaining about a missing space to line up the indentation :rotating_light: :smile:

Reply:
comment line is removed